build.sc

```diff
@@ -1,37 +1,76 @@
 import coursier.Repository
 import coursier.maven.MavenRepository
 import mill._
-import scalalib._
-import scalafmt._
+import mill.scalalib._
+import mill.scalalib.scalafmt._
```

Comment:
? does mill have its own `scalalib`?

Reply:
It is their package for all the scala/java based modules. I would guess this change was just package expansion on the part of the formatter.